test/cases/payments/mediated.test.ts

```diff
@@ -0,0 +1,130 @@
+import { tokenAddresses, toWei } from '../../../src/util/token';
+import mediated from '../../../topologies/mediated.json';
+import setupTestEnvironment from '../../../src';
+import { LuminoTestEnvironment } from '../../../src/types/lumino-test-environment';
+import { LuminoNode } from '../../../src/types/node';
+import { Dictionary } from '../../../src/util/collection';
+import { sleep, verifyChannel } from '../../utils';
+import {
+    SETUP_TIMEOUT,
+    TEARDOWN_TIMEOUT,
+    ChannelState,
+    State,
+    TEST_TIMEOUT,
+} from '../../common';
+
+interface paymentParams {
+    token: string;
+    partner: string;
+    amount: number;
+}
+
+describe('payments mediated', () => {
+    let nodes: Dictionary<LuminoNode>;
+    let env: LuminoTestEnvironment;
+
+    // starting balance for each node
+    const initiatorDeposit = toWei(mediated.channels[0].participant1.deposit);
+    const mediatorInitiatorDeposit = toWei(
+        mediated.channels[0].participant2.deposit
+    );
+    const mediatorTargetDeposit = toWei(
+        mediated.channels[1].participant1.deposit
+    );
+    const targetDeposit = toWei(mediated.channels[1].participant2.deposit);
+
+    const paymentAmount = toWei(1);
```

Comment:
I think is ok but we should start moving all these common values to constants on something called `commons` or something like that to use a reference of it instead of being calling web3 each time and calculating the amount to use. This applies to all the other cases where we use 0, 1 or some common value.

Reply:
I agree we both of you. I like @JONAF2103 's idea.

Reply:
hmm, not sure if that would be too much abstraction, but i'm open to the idea for now. 

i'm willing to review such an implementation if you guys want to include it in some other PR